lib/http/response.rb

```diff
@@ -125,6 +129,15 @@ def cookies
       end
     end
 
+    def chunked?
+      encoding = headers.get(Headers::TRANSFER_ENCODING)
+      case encoding
+        when String then encoding == CHUNKED
+        when Enumerable then encoding.include?("chunked")
+        else false
+      end
```

Comment:
@ixti , you are right, it's a needless case statement. However, consider the [RFC](http://greenbytes.de/tech/webdav/rfc7230.html#message.body.length) and help me figure out point 3:

> If a Transfer-Encoding header field is present in a response and the chunked transfer coding is not the final encoding, the message body length is determined by reading the connection until it is closed by the server. If a Transfer-Encoding header field is present in a request and the chunked transfer coding is not the final encoding, the message body length cannot be determined reliably; the server must respond with the 400 (Bad Request) status code and then close the connection.

Should one tag the response as `chunked?` if it is not the last encoding? I'll add it, unless you feel differently.